kubectl/step2.md

```diff
@@ -1,9 +1,9 @@
-Let’s setup the terminal to help you ease through the command line
-Setup up kubectl completion.
+Let’s setup the terminal to help you ease through using the command line 
```

Comment:
It would be nice in some way to highlight a use case of auto completion more clearly in this step.